cl/alerts/tasks.py

```diff
@@ -31,28 +32,80 @@ def make_alert_messages(
     """Make docket alert messages that can be sent to users
 
     :param d: The docket to work on
     :param new_des: The new docket entries
     :param email_addresses: A list of user email addresses to send to
+    :param recap_email_recipients: The recap_email addresses if needed to send
+    the first case-user notification email.
     :return: A list of email messages to send
     """
+
     case_name = trunc(best_case_name(d), 100, ellipsis="...")
-    subject_template = loader.get_template("docket_alert_subject.txt")
-    subject = subject_template.render(
-        {
-            "docket": d,
-            "count": new_des.count(),
-            "case_name": case_name,
-        }
-    ).strip()  # Remove newlines that editors can insist on adding.
-    email_context = {"new_des": new_des, "docket": d}
+    messages = []
     txt_template = loader.get_template("docket_alert_email.txt")
     html_template = loader.get_template("docket_alert_email.html")
-    messages = []
-    for email_address in email_addresses:
+    subject_template = loader.get_template("docket_alert_subject.txt")
+    subject_context = {
+        "docket": d,
+        "count": new_des.count(),
+        "case_name": case_name,
+        "first_email": False,
+        "auto_subscribe": False,
+    }
+    email_context = {
+        "new_des": new_des,
+        "docket": d,
+        "docket_alert": None,
+        "first_email": False,
+        "auto_subscribe": False,
+    }
+
+    # Concatenate current subscriptions addresses and recap_email_recipients
+    email_addresses_list = list(email_addresses) + recap_email_recipients
+    for email_address in email_addresses_list:
+        if email_address in recap_email_recipients:
+            user_profile = UserProfile.objects.get(recap_email=email_address)
+            # Override user's @recap.email with the actual user's email address
+            email_address = user_profile.user.email
+            docket_alert_exist = DocketAlert.objects.filter(
+                docket_id=d, user_id=user_profile.user.pk
+            ).exists()
+            # If a docket alert exists for this @recap.email user, avoid
+            # sending the first email.
+            if docket_alert_exist:
+                continue
+            first_email = True
+            if user_profile.auto_subscribe:
+                # First time recipient, auto_subscribe True
+                auto_subscribe = True
+                docket_alert = DocketAlert.objects.create(
+                    docket=d, user_id=user_profile.user.pk
+                )
+            else:
+                # First time recipient, auto_subscribe False
+                auto_subscribe = False
+                docket_alert = DocketAlert.objects.create(
+                    docket=d,
+                    user_id=user_profile.user.pk,
+                    alert_type=DocketAlert.UNSUBSCRIPTION,
+                )
+        else:
+            # Not a recap email recipient
```

Comment:
Yes, @recap.email recipients end up here for the following emails if they are subscribed to the case but also this is for not @recap.email users like users that subscribed via Courtlistener page (even if it's their first notification for this case), so I changed the comment to:
`# Options for users that already have an active subscription for this case`

Because a user that subscribed via Courtlistener page doesn't receive a first-time notification they always receive a normal notification.